Dockerfile

```diff
@@ -8,15 +8,18 @@ ARG XX_VERSION=1.6.1
 ARG VPNKIT_VERSION=0.5.0
 
 # DOCKERCLI_VERSION is the version of the CLI to install in the dev-container.
-ARG DOCKERCLI_VERSION=v28.0.1
+ARG DOCKERCLI_VERSION=v28.1.0
```

Comment:
```suggestion
ARG DOCKERCLI_VERSION=v28.1.1
```

Reply:
Ha! I noticed you were working on .1 after I pushed, so knew it was gonna be a race. Will update when I get back to my computer